src/index.ts

```diff
@@ -17,6 +17,24 @@ joplin.plugins.register({
 			}
 		});
 
+		await joplin.settings.registerSettings({
+			"defaultNoteTemplateId": {
+				public: false,
+				type: SettingItemType.String,
+				value: null,
+				label: "Default note template ID"
+			}
+		});
+
+		await joplin.settings.registerSettings({
+			"defaultTodoTemplateId": {
+				public: false,
+				type: SettingItemType.String,
+				value: null,
+				label: "Default to-do template ID"
+			}
+		});
```

Comment:
The point of the new "registerSettings" call (as opposed to "registerSetting") was to limit the number of ipc calls between the plugin and the app, however if you make separate calls, each with only one property, we're back to square one :) So could you group the calls instead?

This should work:

```typescript
await joplin.settings.registerSettings({
	"defaultTodoTemplateId": {
		public: false,
		type: SettingItemType.String,
		value: null,
		label: "Default to-do template ID"
	},
	"defaultNoteTemplateId": {
		public: false,
		type: SettingItemType.String,
		value: null,
		label: "Default note template ID"
	},
});
```

Reply:
Ohk. Noted.